README.md

```diff
@@ -84,6 +84,7 @@
   * [Prerequisites](#prerequisites)
       * [Setup](#setup)
 * [Contributing](#contributing)
+* [App Availability](#App--Availability)
```

Comment:
Should be one dash only to link properly

Reply:
okay will try that

Reply:
> Should be one dash only to link properly

It works thank you so much